module/model-elasticsearch/src/config.ts

```diff
@@ -12,6 +13,7 @@ export class ElasticsearchModelConfig {
   schemaConfig: EsSchemaConfig = {
     caseSensitive: false
   };
+  auth: BasicAuth | ApiKeyAuth;
```

Comment:
This isn't needed here, but we can support it.  What was missing is in the source.ts, when we initClient,

```typescript
    this.client = new es.Client({
        nodes: this.config.hosts,
      ...this.config
    });
```

We should be spreading `this.config.options` and should allow for any ES client options

Reply:
updated PR, I assumed that, but I wasn't sure tbh